app/experimenter/nimbus-ui/src/components/AppLayoutWithExperiment/index.tsx

```diff
@@ -54,10 +68,24 @@ const AppLayoutWithExperiment = ({
     stopPolling,
     review,
   } = useExperiment(slug);
+  const { execute: fetchAnalysis, result: analysis } = useAnalysis();
   const [analysisFetched, setAnalysisFetched] = useState<boolean>(false);
   const status = getStatus(experiment);
 
-  const { execute: fetchAnalysis, result: analysis } = useAnalysis();
+  let redirectPath: string | undefined, getRedirect: string | void;
```

Comment:
Could be good to have a comment here about redirecting based on experiment status, I probably only think that since this component is getting a bit hairy.

Reply:
No that's a good thought, I'll add a note!